hack/dockerfile/install/containerd.installer

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+CONTAINERD_COMMIT=cfd04396dc68220d1cecbe686a6cc3aa5ce3667c # v1.0.2
```

Comment:
We should include the comments from the binary-commits in these files (i.e. update vendor.conf if needed, and vice-versa)

Also (mentioned that somewhere else) would be nice to have a utility to automatically fetch the git-commit from the vendor.conf

Reply:
Updated to add the comments.

The fetch commit thing shouldn't be too hard, but I'd prefer to do it separately.